docs/src/manual/grid.md

```diff
@@ -71,7 +77,48 @@ julia> cells = [
 ```
 
 where each Quadrilateral, which is a subtype of `AbstractCell` saves in the field `nodes` the tuple of node IDs.
-Additionally, the data structure `Grid` can hold node-, face- and cellsets. 
+Each `<: AbstractCell` defines, just as in the example above, the nodes in anti-clockwise ordering.
+This is a convention in Ferrite, which is used to ensure that the transformation from reference space to physical space yields a positive determinant of the Jacobi.
+The function `reference_coordinates` (in `src/interpolations.jl`) defines the expected node ordering by the index of the returned `Vector{Vec{dim,T}}` and is dispatched for each `Interpolation`.
+### Anti-Clockwise Ordering
+Ferrite.jl's ordering w.r.t. nodes, edges and faces of each `<:AbstractCell` or `<:Interpolation`, respectively, can be visualized by `elementinfo` of [FerriteViz.jl](https://github.com/Ferrite-FEM/FerriteViz.jl) with the Makie backend of your choice.
+Below, the anti-clockwise ordering of nodes and faces are shown for different two-dimensional interpolations.
+
+```@example ordering
+import CairoMakie
```

Comment:
Would it make sense to generate this figure statically, to avoid depending on Makie to build the docs.
Looks like the CI-time for docs went up by 3 minutes...

Reply:
I included it not statically to have the ability later to create more figures on the fly, since they bloat quite a bit the git repository. However, if the majority wants them statically tracked by git, I can include it. I thought it's maybe a better approach for the future, since its likely that there will be more and more small pictures in the docs whether its about examples or general docs. What do you prefer @fredrikekre

Reply:
I like Fredrik's approach of the assets folder in the ghpages branch...
That shouldn't bloat the git history...